model/goniometer.py

```diff
@@ -236,9 +244,9 @@ def from_phil(params, reference=None):
                     params, reference
                 )
         else:
-            if params.goniometer.axes is None:
+            if params.goniometer.axis is None and params.goniometer.axes is None:
```

Comment:
Would the following logic be clearer here?
```
            if params.goniometer.axes and len(params.goniometer.axes) > 3:
                goniometer = GoniometerFactory.multi_axis_goniometer_from_phil(params)
            elif params.goniometer.axis or params.goniometer.axes:
                goniometer = GoniometerFactory.single_axis_goniometer_from_phil(params)
            else:
                return None
```

Reply:
:+1: